doc/api/source_map.md

```diff
@@ -0,0 +1,85 @@
+# Source Map
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!--name=source_map-->
+
+Exposes an API for interacting with Node.js' internal cache of source maps.
+
+To enable source map parsing, Node.js must be run with the flag
+[`--enable-source-maps`][], or with code coverage enabled by setting
+[`NODE_V8_COVERAGE=dir`][].
+
+```js
+const source_map = require('source_map');
+```
+
+The APIs and implementation are subject to change at any time.
```

Comment:
I don't think this sentence is necessary given the `Experimental` status notification above.
```suggestion
```